ceremonies/src/lib.rs

```diff
@@ -152,12 +152,12 @@ decl_module! {
                 ok_or("[EncointerCeremonies]: Overflow adding new participant to registry")?;
             if let Some(p) = proof {
                 // we accept proofs from other communities as well. no need to ensure cid
-                ensure!(sender == p.prover_public, "supplied proof is not proving sender");
-                ensure!(p.ceremony_index < cindex, "proof is acausal");
-                ensure!(p.ceremony_index >= cindex-REPUTATION_LIFETIME, "proof is outdated");
+                ensure!(sender == p.prover_public, Error::<T>::ProofSenderMissing);
```

Comment:
it's not missing, its wrong. Rather `WrongProofSubject`

Reply:
Would you prefer the ProofSenderMissing to be renamed to WrongProofSubject or ProofWrongSender?
Using the first would probably require to be used by the outdated and acausal case as well, wouldn't it?

Reply:
`ProofWrongSender` sounds weird to me. If we go with the thinking, what we are talking about the sender/subject, it should be called. `WrongProofSubject`.